include/seqan3/range/container/small_vector.hpp

```diff
@@ -333,13 +333,13 @@ class small_vector
         return &data_[sz];
     }
 
-    //!\copydoc small_vector::end().
+    //!\copydoc seqan3::small_vector::end()
```

Comment:
```suggestion
    //!\overload
```

?

Reply:
I changed it to always use copydoc because the documentation looks better

Reply:
Maybe that's something we should discuss? :man_shrugging:

Reply:
right now it's done using copydoc everywhere. I think the documentation looks nicer that way. but yes I can add it to the next strategy meeting. doesn't hurt